reposettings.py

```diff
@@ -169,6 +169,41 @@ def set(repo: Repository.Repository, config):
                 newsettings['dismiss_stale_reviews'] = bool(rules['dismiss-stale-reviews'])
             if 'required-review-count' in rules:
                 newsettings['required_approving_review_count'] = int(rules['required-review-count'])
+            if 'dismiss-pull-request-reviews' in rules:
+                if 'users' in rules['dismiss-pull-request-reviews']:
+                    newsettings['dismissal_users'] = rules['dismiss-pull-request-reviews']['users']
+                if 'teams' in rules['dismiss-pull-request-reviews']:
+                    newsettings['dismissal_teams'] = rules['dismiss-pull-request-reviews']['teams']
+                if 'apps' in rules['dismiss-pull-request-reviews']:
+                    newsettings['dismissal_apps'] = rules['dismiss-pull-request-reviews']['apps']
+            if 'allow-bypass-pull-request-reviews' in rules:
+                if 'users' in rules['allow-bypass-pull-request-reviews']:
+                    newsettings['users_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['users']
+                if 'teams' in rules['allow-bypass-pull-request-reviews']:
+                    newsettings['teams_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['teams']
+                if 'apps' in rules['allow-bypass-pull-request-reviews']:
+                    newsettings['apps_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['apps']
+            if 'push-restrictions' in rules:
+                if 'users' in rules['push-restrictions']:
+                    newsettings['user_push_restrictions'] = rules['push-restrictions']['users']
+                if 'teams' in rules['push-restrictions']:
+                    newsettings['team_push_restrictions'] = rules['push-restrictions']['teams']
+                if 'apps' in rules['push-restrictions']:
+                    newsettings['app_push_restrictions'] = rules['push-restrictions']['apps']
+            if 'enforce-admins' in rules:
+                newsettings['enforce_admins'] = bool(rules['enforce-admins'])
+            if 'restrict-pushes-create-matching-branches' in rules:
+                newsettings['block_creations'] = bool(rules['restrict-pushes-create-matching-branches'])
```

Comment:
Exactly the same as this comment: https://github.com/txqueuelen/reposettings/pull/11/files#r1176131382

API asks for `block_creations` and this is in the UI:
![GitHub screenshot where you can read: Restrict pushes that create matching branches. Only people, teams, or apps allowed to push will be able to create new branches matching this rule.](https://user-images.githubusercontent.com/53659978/234209427-e65ce212-43ee-4f8f-a782-31918ad97bdc.png)

Reply:
Solved in [#0e24cca](https://github.com/txqueuelen/reposettings/pull/11/commits/0e24ccaecb00a4d1f421b4f42ec3d8aa6c27b704)